src/ansys/templates/python/pyace_pckg/cookiecutter.json

```diff
@@ -0,0 +1,24 @@
+
+{
+  "project_name": "project",
+  "__project_name_slug": "{{ cookiecutter.project_name | slugify(separator=('_')) }}",
+  "library_name": "library",
+  "ci_cd_platform": ["GitHub", "Azure DevOps"],
+  "enable_docker": ["No", "Yes"],
+  "enable_full_observability_stack": ["No", "Yes"],
+  "copyright": "None",
```

Comment:
I know these templates are expected to be used by ACE developers. Which is the license policy on the software? Just curious about this. 

I assume none of this code may be published, so this is why a "copyright" is provided, right?

Reply:
People from ACE can produce two type of project:

- A consulting project for a specific customer. In such case, their is often an IP aggreement set through the frame agreement or the statement of work. An more often, the customer claim the property. Therefore, it's their copyright
- Reusable component that can be used in multiple customer project. In this case either it is Ansys copyright and fully internal or (since the emergency of PyAnsys initiative) open source (e.g MIT license, etc.)